core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -336,13 +336,13 @@
           + "scan is switched to that local file. We can not switch to the minor compacted file because it may have been modified by iterators. The file "
           + "dumped to the local dir is an exact copy of what was in memory."),
   TSERV_BULK_PROCESS_THREADS("tserver.bulk.process.threads", "1", PropertyType.COUNT,
-      "The master will task a tablet server with pre-processing a bulk file prior to assigning it to the appropriate tablet servers. This configuration"
+      "The master will task a tablet server with pre-processing a bulk import file prior to assigning it to the appropriate tablet servers. This configuration"
```

Comment:
Do we want to specify "bulk import RFile" here (and possibly in other places), since RFile is the file type for bulk-imported files, or is "bulk import file" sufficient?

Reply:
`bulk import RFile` makes sense to me.  I will update.